src/foam/u2/dialog/NotificationMessage.js

```diff
@@ -20,111 +20,204 @@ foam.CLASS({
+      position: absolute;
+      top: 0;
     }
     @keyframes fade {
       0% { opacity: 0; }
       10% { opacity: 1; }
       80% { opacity: 1; }
       100% { opacity: 0; }
     }
-    ^error-background{
-      background: #f4cccc;
-      border: 1px solid #f33d3d;
+    ^status-icon {
+      width: 20px;
+      height: 20px;
+      margin-right: 16px;
+      vertical-align: middle;
     }
-    ^ .close-x{
-      right: 5px;
-      top: 10px;
+    ^content {
+      display: inline-block;
+      vertical-align: middle;
+      font-family: IBMPlexSans;
+      font-size: 16px;
+      font-weight: 600;
+      color: #19402e;
+      letter-spacing: normal;
     }
-    ^ .foam-u2-ActionView-close{
-      width: 30px;
-      height: 30px;
-      position: absolute;
-      left: 0px;
-      top: -5px;
-      z-index: 101;
-      opacity: 0.01;
+    ^description {
+      font-family: IBMPlexSans;
+      font-size: 14px;
+      font-weight: normal;
+      line-height: 1.43;
+      letter-spacing: normal;
+      color: #19402e;
+      margin-left: 36px;
     }
-    ^ .close-x {
-      position: absolute;
-      width: 32px;
-      height: 32px;
-      opacity: 0.3;
+    ^error-background {
+      background: #fff6f6;
     }
-    ^ .close-x:hover {
-      opacity: 1;
+    ^warning-background {
+      background: #fdf8de;
     }
-    ^ .close-x:before, .close-x:after {
-      position: absolute;
-      content: ' ';
-      height: 20px;
-      width: 2px;
-      background-color: #333;
+    ^info-background {
+      background: #e5f1fc;
+    }
+    ^error-banner {
+      background: #d9170e;
+    }
+    ^warning-banner {
+      background: #eedc00;
     }
-    ^ .close-x:before {
-      transform: rotate(45deg);
+    ^info-banner {
+      background: #406dea;
     }
-    ^ .close-x:after {
-      transform: rotate(-45deg);
+    ^error-content {
+      color: #631414;
+    }
+    ^warning-content {
+      color: #816819;
+    }
+    ^info-content {
+      color: #202341;
+    }
+    ^link-icon {
+      display: inline-block;
+      margin-top: 2px;
+      vertical-align: middle;
+      margin-right: 0 !important;
+      width: 16px;
+      height: 16px;
+    }
+    ^close-icon {
+      background-image: url("images/round-close-icon.svg");
+      background-size: 12px 12px;
+      cursor:pointer;
+      height: 12px;
+      opacity: 0.5;
+      width: 12px;
+      position: absolute;
+      top: 18;
+    }
+    ^close-icon:hover {
+      opacity: 1;
     }
   `,
 
+  properties: [
+    {
+      class: 'String',
+      name: 'type'
+    },
+    'message',
+    'description'
+  ],
+
   methods: [
+
     function initE() {
       var self = this;
 
+      var img;
+      if ( this.type === 'error' ) {
+        img = 'images/round-error-red.svg';
+      } else if ( this.type === 'warning' ) {
+        img = 'images/baseline-warning-yellow.svg';
+      } else if ( this.type === 'info' ) {
+        img = 'images/round-error-blue.svg';
+      } else {
```

Comment:
what is this else supposed to represent? should the default be 'info'? so the fall through should be to images/round-error-blue.svg?